cstree/src/syntax/text.rs

```diff
@@ -450,4 +457,19 @@ mod tests {
         check(&["{", "abc", "}"], &["{", "123", "}", "{"]);
         check(&["{", "abc", "}ab"], &["{", "abc", "}", "ab"]);
     }
+
+    #[allow(dead_code)]
+    mod impl_asserts {
+        use std::marker::PhantomData;
+
+        use super::*;
+
+        struct NotClone;
+
+        fn assert_copy<C: Copy>(_: PhantomData<C>) {}
+
+        fn test_impls_copy() {
+            assert_copy(PhantomData::<SyntaxText<TokenInterner, SyntaxKind, NotClone>>);
+        }
```

Comment:
And same here:

```suggestion
        fn assert_copy<C: Copy>() {}

        fn test_impls_copy() {
            assert_copy::<SyntaxText<TokenInterner, SyntaxKind, NotClone>>();
        }
```

which then means we don't need `PhantomData` at all ☺️